File: backend/job-application-service/src/main/java/com/jobportal/jobapp/dto/CityDto.java
package com.jobportal.jobapp.dto;

public class CityDto {

    private Integer cid;
    private String cname;
    private Integer sid;

    public CityDto() {
    }

    public CityDto(Integer cid, String cname, Integer sid) {
        this.cid = cid;
        this.cname = cname;
        this.sid = sid;
    }

    public Integer getCid() {
        return cid;
    }

    public void setCid(Integer cid) {
        this.cid = cid;
    }

    public String getCname() {
        return cname;
    }

    public void setCname(String cname) {
        this.cname = cname;
    }

    public Integer getSid() {
        return sid;
    }

    public void setSid(Integer sid) {
        this.sid = sid;
    }
}

File: backend-dotnet/Services/JobPortal.JobApplicationService/DTOs/CityDto.cs
namespace JobPortal.JobApplications.DTOs;

public class CityDto
{
    public int? Cid { get; set; }
    public string? Cname { get; set; }
    public int? Sid { get; set; }

    public CityDto() { }

    public CityDto(int? cid, string? cname, int? sid)
    {
        Cid = cid;
        Cname = cname;
        Sid = sid;
    }
}
